sageopt/coniclifts/constraints/set_membership/pow_cone.py

```diff
@@ -0,0 +1,231 @@
+import numpy as np
+from sageopt.coniclifts.constraints.set_membership.setmem import SetMembership
+from sageopt.coniclifts.base import Expression, ScalarVariable, Variable
+from sageopt.coniclifts.cones import Cone
+from sageopt.coniclifts.operators.norms import vector2norm
+from sageopt.coniclifts.problems.problem import Problem
+
+
+class PowCone(SetMembership):
+    """
+    A power cone constraint is defined with an n-dimensional vector
+    :math:'\\alpha' which is elementwise positive and
+    sums to 1. The cone can be summarized by writing that
+    :math:'(w, z) \\in C_{\\mathrm{power}}' if:
+
+    .. math::
+        \\prod_{i=0}^n w_i^\\alpha_i > \\| z \\|
```

Comment:
I think you need ``w_i^{\\alpha_i}``. The current form might not process the subscript correctly. Also, use ``\\geq`` instead of ``>``.